### PR TITLE
Update to Verified ID HTTP sample

### PR DESCRIPTION
**Pull request by NelsonBlakeN**
Update documentation for Entitlement Management Policy creation to have the correct model for Verified ID issuer

### CODE REVIEW

**Comment by Lauragra** [comment]
@markwahl-msft , can you please verify and approve this PR? Thanks.

File: api-reference/beta/api/entitlementmanagement-post-accesspackageassignmentpolicies.md [hunk 1]
```diff
@@ -887,7 +887,14 @@ Content-type: application/json
   "customExtensionHandlers": [],
   "verifiableCredentialSettings": {
     "credentialTypes":[{
-      "issuers": ["did:ion:EiAlrenrtD3Lsw0GlbzS1O2YFdy3Xtu8yo35W<SNIP>..."],
+      "issuers": [
+        {
```
Comment (FaithOmbongi) [inline api-reference/beta/api/entitlementmanagement-post-accesspackageassignmentpolicies.md:891]: Thank you @NelsonBlakeN for proposing this change.

Unfortunately, from the API schema (see the screenshot below) and the [documentation](https://learn.microsoft.com/en-us/graph/api/resources/verifiablecredentialtype?view=graph-rest-beta), the **issues** object is a String collection type and not a complex type collection like in your change.

image

If the API accepts an object format like you proposed that differs from both the schema and the documentation, then that's a product bug. Please file a product bug at [Microsoft Graph Q&A](https://aka.ms/askgraph for investigation. Please provide a request ID and a sample response showing this deviation between the API response and the schema. The issue will then be investigated.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @NelsonBlakeN - Thank you for proposing this change. However, we're unable to apply it because it differs from the format defined in the documentation and the API schema.

See my inline comment for further guidance.

If you'll be filing an issue at Q&A, you can proceed to close this PR. Thank you.

File: api-reference/beta/api/entitlementmanagement-post-accesspackageassignmentpolicies.md (hunk 1 above)
Comment (NelsonBlakeN) [reply api-reference/beta/api/entitlementmanagement-post-accesspackageassignmentpolicies.md:891]: Hi Faith, thanks for the review. I apologize, we'll need to change the documentation as well. Would it be acceptable to make the doc changes in this same PR as well? Also if that API schema is from our CSDL, we are in the progress of changing that as well. If there is a preferred order or another location, we can take care of it.

Comment (FaithOmbongi) [reply api-reference/beta/api/entitlementmanagement-post-accesspackageassignmentpolicies.md:693]: Hi @NelsonBlakeN - My apologies, I missed your replies due to my inbox rules. I've pinged you on Teams to discuss this further..

**Pull request closed without merging**